lib/common.sh

```diff
@@ -268,23 +269,23 @@ if [[ "${CAPM3RELEASEBRANCH}" = "release-1.3" ]]; then
   export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.3"}
   export BARE_METAL_OPERATOR_TAG="v0.2.0"
   export KEEPALIVED_TAG="v0.2.0"
-  export BMOBRANCH="${BMOBRANCH:-v0.2.0}"
+  export BMOBRANCH="${BMORELEASEBRANCH:-v0.2.0}"
 elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.4" ]]; then
   export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.4"}
   export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.4"}
   export BARE_METAL_OPERATOR_TAG="v0.3.0"
   export KEEPALIVED_TAG="v0.3.0"
-  export BMOBRANCH="${BMOBRANCH:-v0.3.0}"
+  export BMOBRANCH="${BMORELEASEBRANCH:-v0.3.0}"
 elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.5" ]]; then
   export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.5"}
   export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.5"}
   export BARE_METAL_OPERATOR_TAG="v0.4.0"
   export KEEPALIVED_TAG="v0.4.0"
-  export BMOBRANCH="${BMOBRANCH:-v0.4.0}"
+  export BMOBRANCH="${BMORELEASEBRANCH:-release-0.4}"
```

Comment:
We don't have that branch yet btw. 

Should we leave it out of this one, and just use `v0.4.0` like we use `v0.3.0` above, and in another PR later on explicitly move to use release branches?

Btw, why we're using `v0.3.0` when we have `v0.3.1` out?

Reply:
I will put v0.3.1. Didn't notice it. The idea is to merge this once we have the branch. Or do you suggest to merge it with versions now. The jjb PR is also sending release branch name here instead of tag btw. So I have to modify that as well if we decide to merge first and then branch out and add the branch name.

Reply:
I figured we'd do some groundwork now, and then switch to release branches for all supported releases at once, but we can do it this way too.

Reply:
bumped BMO